waiter/src/waiter/process_request.clj

```diff
@@ -189,51 +187,29 @@
     (-> (ex-info message (assoc metrics-map :status status) error)
         (utils/exception->response request))))
 
-(defn http-method-fn
```

Comment:
You're deleting these two functions here because we should have deleted them in #304, right?

Reply:
The net effect is the same where we are not using the function but instead calling `http/request` directly. But this deletion is due to refactoring in `process_request/make-http-request` as part of this PR.